cristin-project/src/test/java/no/unit/nva/cristin/projects/create/CreateCristinProjectHandlerTest.java

```diff
@@ -39,12 +45,17 @@
 import nva.commons.core.Environment;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
 
 class CreateCristinProjectHandlerTest {
 
     public static final String NO_ACCESS = "NoAccess";
     public static final String ILLEGAL_CONTRIBUTOR_ROLE = "illegalContributorRole";
     public static final int FIRST_CONTRIBUTOR = 0;
+    public static final String DRUGSTUDY = "DRUGSTUDY";
```

Comment:
I did consider it, but was not sure if it added any improvement. Do you want me to replace the Sets with enum classes?

Reply:
Use enums that explain what the value is — as they stand, they are a bit meaningless and could be anything, whereas enums will be expressed in the OpenApi. This can also be done in a third PR